includes/payment-methods/class-wc-stripe-upe-payment-method-bancontact.php

```diff
@@ -19,7 +19,7 @@ public function __construct() {
 		parent::__construct();
 		$this->stripe_id            = self::STRIPE_ID;
 		$this->title                = 'Pay with Bancontact';
-		$this->is_reusable          = true;
+		$this->is_reusable          = false;
```

Comment:
Is there any reason why you set `is_reusable` for this payment method and the iDEAL payment method to `false`? These payment methods do support recurring payments. Is this related to anything in #1929 perchance?

Reply:
Yeah, I explained above, but in the middle of a giant comment.

> I have disabled ~EPS~ iDeal and Bancontact because they don't seem to be working as off session methods and don't work for the "Add payment method" page.

> Subscriptions and pre-orders weren't supported for these methods and they're not documented as reusable in the plugin like SEPA and credit card under WooCommerce > Settings > Payments > Automatic Recurring Payments, so I guess that should be ok.

In addition to those issues, Stripe documents it as a [single use payment method](https://stripe.com/docs/payments/bancontact/accept-a-payment#:~:text=Bancontact%20is%20a%20single%20use,the%20payment%20succeeded%20or%20failed.). Although it apparently can be saved as SEPA.

Reply:
OK, cool. Yeah, Bancontact get saved and converted into reusable SEPA payment methods. That's been implemented and is working from the checkout. I explain why I'm pretty sure it's working at checkout and not from the Add payment method page in [this comment](https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1929#issuecomment-925145919), if you're intrigued. 

For what it's worth, I don't mind switching them off here; we can turn them back on in #1929 once that wobbly functionality is resolved.